lib/travis/build/script/csharp.rb

```diff
@@ -48,7 +61,7 @@ def export
         end
 
         def install
-          sh.cmd "nuget restore #{config[:solution]}", retry: true if config[:solution]
+          sh.cmd "nuget restore #{config[:solution]}", retry: true if config[:solution] && !is_mono_2?
         end
 
         def script
```

Comment:
This code is executed unless they override install. The other code is executed unless they override script.